napari/plugins/pypi.py

```diff
@@ -71,7 +75,77 @@ def conda_map() -> Dict[PyPIname, Optional[str]]:
         return json.load(resp)
 
 
-def iter_napari_plugin_info() -> Iterator[Tuple[PackageMetadata, bool]]:
+@lru_cache
```

Comment:
```python
@lru_cache
def pypi_plugin_summaries() -> List[SummaryDict]:
    """Return PackageMetadata object for all known napari plugins."""
    url = "https://npe2api.vercel.app/api/summary"
    with urlopen(Request(url, headers={'User-Agent': _user_agent()})) as resp:
        return json.load(resp)

```

This method can be renamed to something mor egeneral since this endpoint is aggregating infor from different source not only pypi. Also the name of the file should probably be changed to npe2api.py maybe?

url should now be changed to this temporal URL
`https://npe2api-5lai6hg8a-napari.vercel.app/api/extended_summary` and as soon as https://github.com/napari/npe2api/pull/11 ios merged we can change to 
https://github.com/napari/npe2api/pull/11

```python
    url = "https://npe2api.vercel.app/api/summary_extended"
```